starlite/config/compression.py

```diff
@@ -87,11 +83,14 @@ def to_middleware(self, app: "ASGIApp") -> "MiddlewareProtocol":
             A middleware instance
         """
         if self.backend == "gzip":
-            handler = cast(
-                "Type[MiddlewareProtocol]", import_string("starlite.middleware.compression.gzip.GZipMiddleware")
-            )
-        else:
-            handler = cast(
-                "Type[MiddlewareProtocol]", import_string("starlite.middleware.compression.brotli.BrotliMiddleware")
+            from starlite.middleware.compression.gzip import (  # pylint: disable=import-outside-toplevel
+                GZipMiddleware,
             )
-        return handler(app=app, **self.dict())
+
+            return cast("ASGIApp", GZipMiddleware(app=app, **self.dict()))
+
+        from starlite.middleware.compression.brotli import (  # pylint: disable=import-outside-toplevel
+            BrotliMiddleware,
+        )
+
+        return BrotliMiddleware(app=app, **self.dict())
```

Comment:
Does this need the same cast from above?

Reply:
It doesn't. Because we use strong typing, the Starlette stuff needs to be casted.